server/models/device.coffee

```diff
@@ -20,25 +20,19 @@ if process.env.NODE_ENV is "production" or process.env.NODE_ENV is "test"
 
 # Update device in cache
 Device.update = (callback) ->
-    # Retrieve all devices
-    Device.request 'all', (err, devices) ->
+    # Retrieve all accesses
+    client.post "request/access/all/", {}, (err, res, accesses) ->
         cache = {}
         if err?
             logger.error err
             callback err
         else
-            if devices?
+            if accesses?
+                devices = accesses.map (access) -> return access.id
                 # Retrieve all accesses
-                devices = devices.map (device) -> return device.id
-                client.post "request/access/byApp/", {}, (err, res, accesses) ->
-                    if err?
-                        logger.error err
-                        callback err
-                    else
-                        for access in accesses
-                            # Check if access correspond to a device
-                            if access.key in devices
-                                cache[access.value.login] = access.value.token
+                for access in accesses
+                    if access.key in devices
```

Comment:
Var devices is declared in l31 but i was thinking that it could better to name it apps instead of devices because static apps are not necessarily devices. What do you think ?